src/aget_local.erl

```diff
@@ -109,6 +110,11 @@ get(AGetTable, Key, Default) ->
 current_ets(#?MODULE{cur_map = CurMap}) ->
     CurMap.
 
+-spec force_switch(#?MODULE{}) -> #?MODULE{}.
+force_switch(#?MODULE{cur_map = CurMap, prev_map = PrevMap} = State) ->
+    ets:delete_all_objects(PrevMap),
+    State#?MODULE{cur_map = PrevMap, prev_map = CurMap}.
```

Comment:
I was thinking about it but it does not reset the timer. Unless you mean a change that does switch the timer?